### ISSUE TITLE
A4 Feedback Corrections

### ISSUE THREAD

**Issue by rkargul** [remla23-team06/model-training#10]
* Ensure the instructions run on Windows, Mac, and Linux (Poetry)
* Use a data science template from [cookiecutter](https://drivendata.github.io/cookiecutter-data-science/)
* Dataset stored outside the project (stored remotely) (downloaded as part of the pipeline)
* ~Instructions on how to run the pipeline~
* Check layout of the repository
* Different metrics are reported
* It is possible to display metrics across multiple branches and tags
* Different metrics are reported beyond model correctness
* Make sure to remove the back-slashes in `poetry run pylint --load-plugins=dslinter --disable=all --enable=import,unnecessary-iteration-pandas,unnecessary-iteration-tensorflow, nan-numpy,chain-indexing-pandas,datatype-pandas, column-selection-pandas,merge-parameter-pandas,inplace-pandas, dataframe-conversion-pandas,scaler-missing-scikitlearn,hyperparameters-scikitlearn, hyperparameters-tensorflow,hyperparameters-pytorch,memory-release-tensorflow, deterministic-pytorch,randomness-control-numpy,randomness-control-scikitlearn, randomness-control-tensorflow,randomness-control-pytorch,randomness-control-dataloader-pytorch, missing-mask-tensorflow,missing-mask-pytorch,tensor-array-tensorflow, forward-pytorch,gradient-clear-pytorch,pipeline-not-used-scikitlearn, dependent-threshold-scikitlearn,dependent-threshold-tensorflow,dependent-threshold-pytorch --output-format=text:report.txt,colorized --reports=y src/` in the README file
